src/aragon-params.ts

```diff
@@ -0,0 +1,15 @@
+// The Aragon web client expects certain parameters to work locally:
+// - Local testnet node to connect to (testnetPort)
+// - ENS address to resolve names (aragenMnemonic)
+// In order to deploy the Aragon bases a specific gas limit is needed (aragenGasLimit)
+export const testnetPort = 8545
+export const aragenGasLimit = 50e6
```

Comment:
Perhaps we can set a more realistic number here (e.g. 10e6) to avoid someone being confused when they go to a real network.

Reply:
I respected the current settings in @aragon/aragen, which should test first if the bases are deployable with a normal gas limit